change/@ni-nimble-components-99393253-07d1-4ec4-902b-41680bddd36e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Implementation of markdown parser for converting markdown input to rich text content in the viewer",
+  "packageName": "@ni/nimble-components",
+  "email": "123377523+vivinkrishna-ni@users.noreply.github.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
Remaining steps for completing this PR
1. resolve open threads and get Milan's approval
2. you don't need to wait for mollykreis as she's out of office and the changes that brought her in are minor
3. update from main. The Chromatic diff currently includes changes that it shouldn't and I think it's because the branch is out of date
4. take a close look at the Chromatic diffs and either approve or ask a Nimble owner to approve

Reply:
@vivinkrishna-ni Thanks for the background. I'm not sure why Chromatic does that sometimes. But I looked at the new baselines and they look good, so I approved them all.